docs/user/02-logs.md

```diff
@@ -170,45 +174,93 @@ Telemetry Manager supports different types of [Fluent Bit filter](https://docs.f
 
 ### Step 4: Add authentication details from Secrets
 
-Integrations into external systems usually need authentication details dealing with sensitive data. To handle that data properly in Secrets, the LogPipeline supports the reference of Secrets.
+Integrations into external systems usually need authentication details dealing with sensitive data. To handle that data properly in Secrets, the LogPipeline supports the reference of Secrets. At the moment, mTLS and Basic Authentication are supported.
```

Comment:
```suggestion
Integrations into external systems usually need authentication details dealing with sensitive data. To handle that data properly in Secrets, the LogPipeline supports the reference of Secrets. At the moment, mutual TLS (mTLS) and Basic Authentication are supported.
```

Because the `<summary`> uses the full name, I'd also mention the full name here. Alternatively, switch the `<summary`> to "mTLS". In that case, we should still keep the full word when introducing the abbreviation.